hazelcast-jet-core/src/main/resources/hazelcast-jet-default.xml

```diff
@@ -32,12 +32,14 @@
         <!-- receive window size multiplier, only applies to distributed edges -->
        <receive-window-multiplier>3</receive-window-multiplier>
     </edge-defaults>
+
     <metrics>
-        <!-- if metrics collection is enabled -->
-        <enabled>true</enabled>
```

Comment:
Should have been turned to an attribute.

Reply:
It's true by default in the XSD, so it's not needed.